timesketch/lib/utils.py

```diff
@@ -60,7 +60,8 @@ def read_and_validate_csv(path):
             if u'timestamp' not in csv_header and u'datetime' in csv_header:
                 try:
                     parsed_datetime = parser.parse(row[u'datetime'])
-                    row[u'timestamp'] = int(time.mktime(parsed_datetime.timetuple()))
+                    row[u'timestamp'] = str(int(
```

Comment:
reverted.  this came in when rebasing and may have been something jesse added in testing.  so long as it still runs locally i'll leave this out.